transformations/azerty_qwerty_chars_swap/transformation.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class AzertyQwertyCharsSwap(SentenceOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
```

Comment:
Please include `keywords` for the transformation as explained [here](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/docs/keywords.md).

Reply:
Done !